crud/common/utils.lua

```diff
@@ -488,4 +493,35 @@ function utils.flatten_obj_reload(space_name, obj)
     return schema.wrap_func_reload(flatten_obj, space_name, obj)
 end
 
+function utils.format_jsonpath(operand, field_names, field_ids)
+    local ind = operand:find("]")
+    if ind == nil or keydef == nil then -- not jsonpath OR tarantool < 2
```

Comment:
What does `ind` mean?
Are you sure that passing JSON path when Tarantool doesn't support it isn't an error?